docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerImagePullStrategy.java

```diff
@@ -0,0 +1,56 @@
+package com.nirima.jenkins.plugins.docker;
+
+/**
+ * @author Kanstantsin Shautsou
+ */
+public enum DockerImagePullStrategy {
+
+    PULL_ALWAYS("Pull all images every time") {
+        @Override
+        public boolean pullIfNotExists(String imageName) {
+            return true;
+        }
+
+        @Override
+        public boolean pullIfExists(String imageName) {
+            return true;
+        }
+    },
+    PULL_LATEST("Pull once and update latest") {
+        @Override
+        public boolean pullIfNotExists(String imageName) {
+            return true;
+        }
+
+        @Override
+        public boolean pullIfExists(String imageName) {
+            return imageName.endsWith(":latest");
+        }
+    },
+    PULL_NEVER("Never pull") {
```

Comment:
Use case it up to user, just implement ability to not pull any images, i.e. somebody on mobile on dial-up